image_grab_test.py

```diff
@@ -0,0 +1,40 @@
+# This file is used to grab images from the camera when the respective keys on the keyboard are
+# pressed. This is used to get test images from the webcam and not intended for use in the actual
+# competition
+
+import cv2
+
+# use the default camera
+input = cv2.VideoCapture(0)
```

Comment:
We should capture images using the same pipeline as the robot will, i.e. VideoLiveGenerator.

Reply:
Thanks for the notice. I have updated this file to use VideoLiveGenerator.